Cargo.toml

```diff
@@ -41,6 +41,7 @@ members = [
   "pallets/connectors",
   "pallets/connectors-gateway",
   "pallets/connectors-gateway/routers",
+  "pallets/connectors-gateway/connectors-gateway-axelar-precompile",
```

Comment:
Nit - maybe rename that folder to `precompile`?

Reply:
named to axelar-gatway-precompile